performance-tests/bench/worker/SetCftParametersAction.cpp

```diff
@@ -156,6 +157,8 @@ void SetCftParametersAction::do_set_expression_parameters()
       }
 #endif
     }
+    last_scheduled_time_ += OpenDDS::DCPS::TimeDuration(set_period_);
```

Comment:
Maybe turn these member `ACE_Time_Value`s into `TimeDuration`s since they're not being passed directly to ACE anymore?